draft-ietf-quic-transport.md

```diff
@@ -4127,10 +4127,12 @@ UDP datagrams MUST NOT be fragmented at the IP layer.  In IPv4
 ({{!IPv4=RFC0791}}), the DF bit MUST be set if possible, to prevent
 fragmentation on the path.
 
-Datagrams are required to be of a minimum size under some conditions.  However,
-the size of the datagram is not authenticated.  Therefore, an endpoint MUST NOT
-close a connection when it receives a datagram that does not meet size
-constraints, though the endpoint MAY discard such datagrams.
+QUIC sometimes requires datagrams to be larger than a minimum size; see
+{{version-negotiation}} as an example. However, the size of a datagram is not
```

Comment:
I think this sentence is incorrect. The intent of the paragraph is to state that receivers MUST NOT close a connection when it receives a datagram of X bytes carrying a QUIC packet, and if that QUIC packet being carried is required to be sent in a datagram of more htan X bytes.

So maybe something like: _QUIC sometimes requires datagrams to be no smaller than certain size; see {{validate-handshake}} as an example._

Reply:
Or "at least"